var/spack/repos/builtin/packages/cray-mpich/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class CrayMpich(Package):
+    """Cray's MPICH is a high performance and widely portable implementation of
+    the Message Passing Interface (MPI) standard."""
+
+    homepage = "https://docs.nersc.gov/development/compilers/wrappers/"
+    has_code = False    # Skip attempts to fetch source that is not available
+
+    maintainers = ['haampie']
+
+    version('7.7.16')
+    version('7.7.15')
+    provides('mpi')
```

Comment:
I think this also needs to `provides('mpi@3')`

Reply:
I've added `@3:`, because I confused about what `@3` means when I read it. Version '3' as an exact match, or 3.x.y for x>=0 and y>=0?

Reply:
Ok

For reference, `@3` satisfies `@3.z.z:3.develop`, which are the lowest and highest possible versions starting with `3.`.